appholder/src/main/java/com/android/mdl/app/document/DocumentDatabase.kt

```diff
@@ -30,7 +30,7 @@ abstract class DocumentDatabase : RoomDatabase() {
 
     companion object {
 
-        private const val DATABASE_NAME = "document-db"
+        private const val DATABASE_NAME = "mdoc-document-database"
```

Comment:
This will break existing installs, right?

Is there really a great reason to rename this?

Reply:
Yeah, it's intentional b/c we're also changing where the credential data is stored.